public/locales/en/translation.json

```diff
@@ -482,6 +482,7 @@
       "datasetNotCreated": "Dataset hasn't been created",
       "datasetNotUpdated": "Dataset hasn't been updated",
       "twingraphNotCreated": "A problem occurred during twingraph creation",
+      "twingraphStatusPollingFailed": "A problem occurred when querying the status of dataset {{datasetDetails}}",
```

Comment:
English and french translations seem to not be exactly the same. Maybe use "when updating the status" instead of "querying" ?

Reply:
I have no suggestion en français. :relieved: